_posts/en/newsletters/2019-08-07-newsletter.md

```diff
@@ -0,0 +1,144 @@
+  payment will require coordination between multiple nodes and so
+  requires modification of the Lightning specification.
+
+## Bech32 sending support
+
+*Week 21 of 24 in a [series][bech32 series] about allowing the people
+you pay to access all of segwit's benefits.*
+
+{% include specials/bech32/21-brd.md %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #15878][] changes the `walletcreatefundedpsbt` to signal
+  [BIP125][] Replace-by-Fee (RBF) if the wallet is configured to use RBF
+  (the configuration option `walletrbf`).
+
+- [Bitcoin Core #16394][] changes the `createwallet` RPC so that if an
+  empty string is passed for the passphrase parameter, an unencrypted
+  wallet will be created (and a warning printed).
+
+- [LND #3184][] adds a watchtower client sub-server to LND that replaces
+  the private watchtower implementation added for the most recent
+  release.
+
+- [LND #3164][] creates a new database into which information about the
+  past 1,000 payments is stored.  (The default of 1,000 can be changed.)
+  On upgrade, details about previous payments will be populated into
+  this database from LND's lower-level HTLC-tracking database.
+
+- [LND #3359][] adds a `ignore-historical-filters` configuration option
+  that will cause LND to to ignore a `gossip_timestamp_filter` sent from
+  a peer.  That filter allowed peers to request all announcements that would've
+  been gossiped during an earlier date range.  By ignoring requests for
+  the filter, LND doesn't need to keep those historic announcements in
+  memory or send them to peers, lowering its memory and bandwidth
+  usage.  The ignore option currently defaults to False, so there's no
+  change in default LND behavior, but a commit comment notes "down the
+  road, the plan is to make this default to True."
+
+- [C-Lightning #2771][] allows plugins to indicate whether or not they
+  can be started and stopped during runtime rather than just being
+  started on init and stopped on shutdown.  This information is used by
+  a new `plugin` command that allows users to perform these runtime
+  stops and starts.
+
+- [C-Lightning #2892][] now always runs plugins from the Lightning
+  configuration directory, reducing inconsistencies between different
+  installs and making it easy for plugins to store and access data
+  within that directory.
+
+- [C-Lightning #2799][] extends the `listforwards` RPC with a
+  `payment_hash` field to allow the user to find additional information
+  about the HTLC.  Additionally, the PR provides a new `forward_event`
+  notification for plugins that alerts when an HTLC has been offered,
+  settled, failed by both parties, or failed locally (unilaterally).
```

Comment:
I suggest reversing the order of these (new `forward_event` notification then change to `listforwards` rpc). The motivation given for the PR was the `forward_event` notification.